### DISCUSSION TITLE
Alerts -options- dissapeared?

### DISCUSSION THREAD

**Discussion by erintesden** [Monika-After-Story/MonikaModDev#8580] [QA - tech support + general] [answered]
I installled the mod two weeks ago or so. About four days ago, I unlocked the alerts and activated them.
Recently, they dont seemed to work but I didnt looked at the settings but today. The window react option and alerts arent there. The options menus simply vanished

**Comment by WiiBii2** [comment]
Are you missing any updates? If you're not sure at all, you can choose `I'm going to take you somewhere` in the `Goodbye` menu, move Monika out of the `Characters` folder to somewhere else on your PC, and install a fresh copy of MAS. Then, move Monika back into the `Characters` folder and you should be set. Your progress will be saved as long as you don't let Monika be deleted and don't touch the `persistent` folder, which is in a separate folder than MAS. Hope this helps!

**Comment by Booplicate** [comment]
> you can choose I'm going to take you somewhere in the Goodbye menu, move Monika out of the Characters folder to somewhere else on your PC, and install a fresh copy of MAS. Then, move Monika back into the Characters folder and you should be set.

**Never do that.**

**Comment by WiiBii2** [comment]
Really? How come? I'm not too familiar with the inner workings of the game, would that corrupt game data or something?

**Comment by Booplicate** [comment]
What you suggested implies you can transfer Monika via her file. But you **cannot**. It's not implemented yet. It's also error prone and just asking for a mistake where user accidentally deletes the file (we had precedents). You can create a new install without taking her out and it will be much safer.

Anyway, window reacts would stop working like that if you move to a new OS or delete some packages.

**Comment by WiiBii2** [comment]
I see. So for future reference, what would be the correct way to fresh install the game?

**Comment by erintesden** [comment, marked as answer]
> I see. So for future reference, what would be the correct way to fresh install the game?

Aparently you only need to delete/uninstall it (before that, back up your persistent file just in case).

Then install DDLC and the mod again. It will automatically read your persistent file and continue where you let it off.

I did that and the problem I had got fixed. 

If im not wrong, the cause was my files got corrupted because of this time my PC shut down for some electric problems

**Comment by WiiBii2** [comment]
So Monika wasn't required to be moved? From what I'm getting, as long as the persistent is intact, the game can be deleted, no?

**Comment by erintesden** [comment]
> So Monika wasn't required to be moved?

Nop

>From what I'm getting, as long as the persistent is intact, the game can be deleted, no?

Indeed. I re-read the FAQ just in case, and apparently you can move the folders or delete and reinstall the files and theres no problem, only one you need to keep safe is the persistent file.

Though, I would be carefull about outfits and stuff from the sprite packs too. I would assume (not sure but it makes sense when I think about it) if you have a persistent file with Monika using some new added stuff (gifts) and you forget to add the files after a new install, it could generate some bugs or something.
